python/GafferUI/EditScopeUI.py

```diff
@@ -233,12 +233,16 @@ def __navigationMenuDefinition( self ) :
 			)
 			return result
 
-		if editScope.processors() :
-			for processor in editScope.processors() :
+		nodes = editScope.processors()
+		nodes.extend( self.__userNodes( editScope ) )
+
+		if nodes :
+			for node in nodes :
+				path = node.relativeName( editScope ).replace( ".", "/" )
```

Comment:
Do we have a use case for nested nodes?

Reply:
It'd been mentioned when going through production templates the other day, so felt like it was just easier to support it now, than revisit again again.